hop/density.py

```diff
@@ -502,8 +502,8 @@ def current_coordinates():
         h[:],edges[:] = numpy.histogramdd(coord, bins=bins, range=arange, normed=False)
         grid += h  # accumulate average histogram
         pm.echo(ts.frame, len(coord))
-    n_frames = u.trajectory.n_frames / u.trajectory.skip
-    grid /= float(n_frames)
+    #n_frames = u.trajectory.n_frames / u.trajectory.skip
+    grid /= float(u.trajectory.n_frames)
```

Comment:
This should depend on how many frames were actually used, i.e., take into account `step` (if we use it).